app/lib/presenter/complaint.rb

```diff
@@ -47,8 +47,8 @@ def customer_data
         'Customer.Postcode': submission_answers.fetch(:postcode, ''),
         'Customer.Email': submission_answers.fetch(:email_address, ''),
         'Customer.Phone': submission_answers.fetch(:phone, ''),
-        'Impact': submission_answers.fetch(:impact, ''),
```

Comment:
Bear with - not sure these linter changes are actually desirable

Reply:
Seems its fine actually - the tests expect this output without quotes so they must not matter outside of style in this case